package.json

```diff
@@ -374,6 +375,7 @@
     "lottie-web": "^5.12.2",
     "luxon": "^3.2.1",
     "nanoid": "^3.3.8",
+    "picomatch": "^4.0.2",
```

Comment:
How important is using `picomatch`, and how much more code is it to use a package we already use?

Reply:
It's needed, the library is also pretty lightweight. I don't think we're doing any glob pattern matching anywhere.

Reply:
I was trying to understand this... I guess it's because CODEOWNERS is written with globs?

Reply:
Yessir